pkg/deploy/lattice/target_group_manager.go

```diff
@@ -30,6 +31,18 @@ func NewTargetGroupManager(cloud lattice_aws.Cloud) *defaultTargetGroupManager {
 	}
 }
 
+func TG2LatticeTGName(targetGroup *latticemodel.TargetGroup) string {
```

Comment:
I think something like `getLatticeTGName()` should be enough, the name does not look straightforward and we do not need initial caps. Or we could make this as a method of `TargetGroup` (it may not look clean though, just presenting an option here).